README.md

```diff
@@ -32,6 +32,19 @@ You can also install *Stretchly* with [Homebrew](https://brew.sh/) by running `b
 
 If you're using [Alfred](https://www.alfredapp.com) on macOS you can use this [Alfred Workflow](https://github.com/KingOfSpades/stretchFred) to interact with Stretchly.
 
+#### macOS Apple Silicon / M1
+
+*Stretchly* is also available as a native M1 application. Currently, it's not published to Homebrew, so you need to download
```

Comment:
I didn't know how you have your Homebrew cask configured, so I added this comment. Perhaps `electron-builder` automatically publishes both versions for you? I'm not sure.

Reply:
Homebrew version is added by community. Looks like they support m1 as well, but cask would need to be updated, one new Stretchly with m1 is out; https://github.com/Homebrew/homebrew-cask/commit/64a4f75db137589efcd5723ff608b99cb3160983

Reply:
Sounds good - I can have a look at this after merge / first ARM-supported release.